plugins/woocommerce/src/Internal/ProductAttributes/GlobalAttributeVariationCleanup.php

```
@@ -0,0 +1,229 @@
+
+	/**
+	 * Schedule variation cleanup after a global attribute is deleted.
+	 *
+	 * @internal
+	 *
+	 * @param int    $attribute_id Attribute ID.
+	 * @param string $name         Attribute name.
+	 * @param string $taxonomy     Attribute taxonomy name.
+	 * @return void
+	 */
+	public function handle_woocommerce_attribute_deleted( $attribute_id, $name, $taxonomy ): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute_id'. (undefined)

(UnusedFormalParameter)